pkg/pack.sh

```diff
@@ -44,7 +71,9 @@ if [ "$(uname -s)" == "Darwin" ]; then
     __targets_param="TargetsOSX=true"
 else
     __targets_param="TargetsLinux=true"
-    init_distro_name
+    if [ -z $__distro_rid ]; then
```

Comment:
When do you expect `--rid` to not be specified? It might be better to just error out here.